Pod/Classes/StackViewHelpers/UIStackView+Helpers.swift

```diff
@@ -28,6 +28,15 @@ public extension UIStackView {
             addArrangedSubview($0)
         }
     }
+    
+    func removeAllArrangedSubviews() {
+        arrangedSubviews.forEach {
+            self.removeArrangedSubview($0)
+            NSLayoutConstraint.deactivate($0.constraints)
```

Comment:
Wrote my previous comment on my phone. This is what I'm saying:

```suggestion
```

You can remove those lines because the last line implies them.